.goreleaser.yml

```diff
@@ -16,7 +16,6 @@ builds:
     - linux
     goarch:
     - amd64
-    - arm
```

Comment:
I think we want to keep Linux arm. See the scenario described by @amcginlay in https://github.com/jetstack/jsctl/issues/51 - if I understand correctly a Linux VM on Apple Silicon too would be arm

Reply:
I think that is arm64, not sure where arm is used

Reply:
ahh my bad, ignore
can't imagine anyone using it on Rasberry PI tbh